projects/cashmere/src/lib/popover/popoverContent.component.ts

```diff
@@ -1,130 +1,131 @@
 import {Component, ElementRef, EventEmitter, HostListener, OnDestroy, Renderer2, ViewChild} from '@angular/core';
 import Popper from 'popper.js';
-import {Placements, PopperContentOptions, Triggers} from './popover.model';
+import {PopperContentOptions} from './popover.model';
 
```

Comment:
Sounds like a usage markdown file would be better

Reply:
I am with @jtrujill a extended usage markdown would be a great addition, especially as their are many scenarios that aren't covered by basic documentation. I can create an issue to expand it

Reply:
@benjanderson if you need help with that usage doc just contact me and I'll give you the rundown. It's very easy. You just need to modify `src/app/core/document-items.service.ts` so that the popover component has a `usageDoc: true` defined. You also place the markdown file at the root of the components directory

Reply:
That's fine, I was mostly saying that this looked kind of empty, so I wasn't sure if we should say something about what it is.  But I defer to you all as to whether that's needed or not.

<img width="492" alt="screen shot 2018-08-06 at 4 20 56 pm" src="https://user-images.githubusercontent.com/22795893/43744705-8284918c-9998-11e8-9600-3fcfe1868f8f.png">

Reply:
I will take a look